include/cxx_argp_parser.h

```diff
@@ -0,0 +1,246 @@
+// Header-only, modern C++ argument parser based on ARGP
+//
+// Copyright (C) 2018-2019 Patrick Boettcher <p@yai.se>
+//
+// SPDX-License-Identifier: LGPL-3.0
```

Comment:
This is LGPL-3, this isn't compatible with the main licence is it?

Reply:
oops, didn't notice the v3 part, I'll make this a submodule then